components/molecule/select/src/components/SingleSelection.js

```diff
@@ -87,7 +88,7 @@ const MoleculeSelectSingleSelection = forwardRef(
           size={selectSize}
           tabIndex={tabIndex}
         >
-          <AtomInput ref={forwardedRef} inputMode={inputTypes.NONE} noBorder />
+          <AtomInput aria-describedby={ariaDescribedBy} ref={forwardedRef} inputMode={inputTypes.NONE} noBorder />
```

Comment:
Correct me if Im wrong, but what if you use the spread props approach here? I think the`AtomInput` accepts whatever prop you throw at it and pass it down to it's `Component`. Then you wouldn't have to specifically pass down the `aria-describedby` prop
```suggestion
          <AtomInput ref={forwardedRef} inputMode={inputTypes.NONE} noBorder {...props} />
```